frontend/src/bundles/common/components/sub-navigation/components/sub-nav-item.tsx

```diff
@@ -0,0 +1,51 @@
+import { NavLink } from 'react-router-dom';
+
+import { getValidClassNames } from '~/bundles/common/helpers/helpers.js';
+
+type Properties = {
+    label: string;
+    to: string;
+    isActive: boolean;
```

Comment:
I think that it does not neccessary to provide isActive and compare path with hook, because in NavLink already is className active 
![image](https://github.com/BinaryStudioAcademy/bsa-winter-2023-2024-lime/assets/112903531/132d3c1d-97d9-4c2f-bd30-1edc8c695949)
and you can change a bg color with this class

Reply:
I don't how I can implement it with tailwindcss because I need pass styles for children elements **span** and **p** but not in **a** element

Reply:
NavLink should pass these as props to children
so you should be able to do something like this
`<NavLink>
  { ({ isActive }) => (
    <>
      Style elements based on isActive here
    </>
  ) }
</NavLink>`

Reply:
Thanks, I will do it